packages/ui/misc/ResponsiveImage/shopifyUtils.ts

```diff
@@ -10,10 +10,14 @@ const CDN_HOSTNAMES = [
 const BASE_IMAGE_SIZE = 352;
 
 export function isShopifyImage(src: string) {
-   const newSrc = new URL(src);
-   return CDN_HOSTNAMES.some((allowedHostname) =>
-      newSrc.hostname.endsWith(allowedHostname)
-   );
+   try {
+      const url = new URL(src);
+      return CDN_HOSTNAMES.some((allowedHostname) =>
+         url.hostname.endsWith(allowedHostname)
+      );
+   } catch (e) {
+      return false;
+   }
```

Comment:
?

Reply:
I've opened [a separate issue](https://github.com/iFixit/react-commerce/issues/1365) to describe the problem.
The try catch is a temporary solution to avoid making large changes to the loader selection logic in this PR.

Reply:
Got it